src/test/java/com/weaponcharges/ConfigGenerator.java

```diff
@@ -4,7 +4,7 @@
 
 public class ConfigGenerator
 {
-	static int i = 0;
+	static int i = -1;
```

Comment:
why did you change this.

Reply:
This is a change made for `ConfigGenerator.java`'s new functionality (support for skipping serp display style config item) and does not break anything. It works as intended.